src/Clauses/OptionalMatchClause.php

```diff
@@ -63,7 +69,7 @@ protected function getSubject(): string
     {
         return implode(
             ", ",
-            array_map(fn(StructuralType $pattern): string => $pattern->toQuery(), $this->patterns)
+            array_map(fn( $pattern ): string => $pattern->toQuery(), $this->patterns)
```

Comment:
Please format this using the PSR-2 coding standard (see https://www.php-fig.org/psr/psr-2/#43-methods)